plutus-tx-plugin/src/PlutusTx/Compiler/Type.hs

```diff
@@ -263,19 +265,6 @@ getConstructors tc = do
         Just constrs -> pure constrs
         Nothing      -> throwSd UnsupportedError $ "Cannot construct a value of type:" GHC.<+> GHC.ppr tc GHC.$+$ ghcStrictnessNote
 
--- | Get the constructors of the given 'Type' (which must be equal to a type constructor application) as PLC terms instantiated for
--- the type constructor argument types.
-getConstructorsInstantiated :: Compiling uni fun m => GHC.Type -> m [PIRTerm uni fun]
-getConstructorsInstantiated t = withContextM 3 (sdToTxt $ "Creating instantiated constructors for type:" GHC.<+> GHC.ppr t) $ case t of
-    (GHC.splitTyConApp_maybe -> Just (tc, args)) -> do
-        constrs <- getConstructors tc
-
-        forM constrs $ \c -> do
-            args' <- mapM compileTypeNorm args
-            pure $ PIR.mkIterInst () c args'
-    -- must be a TC app
-    _ -> throwSd CompilationError $ "Cannot construct a value of a type which is not a datatype:" GHC.<+> GHC.ppr t
-
```

Comment:
I decided to remove it because it's unused.

Reply:
Huh, I guess we didn't get a warning because it's exported.